test/functional/view-entity/view-dependencies/view-entity-dependencies.test.ts

```diff
@@ -10,7 +10,8 @@ import { ViewA } from "./entity/ViewA"
 import { ViewB } from "./entity/ViewB"
 import { TestEntity } from "./entity/Test"
 
-describe("views dependencies", () => {
+// TODO: Implement topological sorting for view dependencies https://github.com/typeorm/typeorm/issues/8240
+describe.skip("views dependencies", () => {
```

Comment:
I noticed this test started to fail after this change, and for very good reason: this feature was not implemented correctly.

Since the purpose of this PR is to simply fix the file names, I will leave the feature to be fixed separately.